changelog.md

```diff
@@ -12,7 +12,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 -->
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+## [2.3.4] - 2023-02-25
+### Added
+- Support for asynchronous TCP and RTU/Serial servers and clients, through the `asynchronous` package, see #5
+### Fixed
```

Comment:
please add one empty line before new headers

```suggestion

### Fixed
```

Reply:
Thanks for pointing it out - I didn't notice the discrepancy between other changelog headers. Will change it